model_templates/python3_unstructured/custom.py

```diff
@@ -19,12 +22,32 @@ def score_unstructured(model, data, query, **kwargs):
     mlops = kwargs.get("mlops")
     print(f"MLOps supported: {mlops is not None}")
 
+    headers = None
+    if "headers" in kwargs:
+        headers = kwargs["headers"]
+        print("Incoming request headers: ", headers)
     print("Incoming query params: ", query)
+
+    if headers and "multipart/form-data" in headers.get("Content-Type"):
+        # For more information refer to:
+        # https://werkzeug.palletsprojects.com/en/2.2.x/http/#module-werkzeug.formparser
+        environ = {
+            "wsgi.input": BytesIO(data),
+            "CONTENT_LENGTH": headers.get("Content-Length"),
+            "CONTENT_TYPE": headers.get("Content-Type"),
+            "REQUEST_METHOD": "POST",
+        }
+        stream, form, files = werkzeug.formparser.parse_form_data(environ)
+        filestorage = files.get("filekey")
+        data = filestorage.stream.read()
+
     if isinstance(data, bytes):
-        data = data.decode("utf8")
+        text_data = data.decode("utf8")
+    else:
+        text_data = data
```

Comment:
nit (should include line 44, but github do not support suggestion on deleted lines):
```suggestion
        text_data = data.decode("utf8") if isinstance(data, bytes) else data
```